ecosystem/typescript/sdk/package.json

```diff
@@ -53,6 +53,7 @@
   "dependencies": {
     "@noble/hashes": "1.1.3",
     "@scure/bip39": "1.1.0",
+    "aptos": "^1.12.0",
```

Comment:
hmm that doesnt make sense, we dont need the `aptos` package as a dependency in the aptos sdk